packages/components/src/tag/tag.tsx

```diff
@@ -17,17 +21,34 @@ const textSizes: Record<NonNullable<Props['size']>, TextProps['size']> = {
   '24': 13,
 }
 
+const iconSizes: Record<NonNullable<Props['size']>, IconProps['size']> = {
+  '32': 20,
+  '24': 12,
+}
+
 const Tag = (props: Props) => {
-  const { size, emoji, label, selected, disabled } = props
+  const { size, icon, label, selected, disabled } = props
+
+  const renderIcon = () => {
+    if (!icon) {
+      return null
+    }
+
+    if (typeof icon === 'string') {
+      return <Text size={textSizes[size]}>{icon}</Text>
```

Comment:
I was watching the figma file and i understand what you're doing with the emoji. You're using the text emoji from Apple in this example but we're going the use our custom svg emoji. I think. I think it's ok now.

Reply:
The `SVGs` do not fall into this condition.